modules/lang-painless/src/main/java/org/elasticsearch/painless/PainlessExecuteAction.java

```diff
@@ -193,6 +318,10 @@ public static SupportedContext fromId(byte id) {
                 switch (id) {
                     case 0:
                         return PAINLESS_TEST;
```

Comment:
We already have a unique identifier for each context (the name). Could we use that instead of creating an enumeration here that will need to be updated for every new context?

Reply:
Good point, let me try that.